### PR TITLE
feat: logs,status,export

### PR DESCRIPTION
**Pull request by wartraxx51**
Here's a PR that allows reading Docker logs, extracting those logs into an archive, and also displaying a brief status.
I named it monitoring.py, but I had other ideas like diag, debug, or obs (for observability). Feel free to let me know which you prefer.

Here are a few screenshots:
image
image

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -202,7 +202,7 @@ services:
       timeout: 5s
       retries: 5
 
-  infrahub-server:
```
Comment (BaptisteGi) [inline docker-compose.yml]: Heuu, last time we changed that we faced many side effects, don't know if we want to do that again

**Comment by BaptisteGi** [comment]
@wartraxx51 If your work is done on this one might be worth removing the draft status

**Comment by wartraxx51** [comment]
I had a meeting yesterday about my PR with @dgarros and @fatih-acar  on the topic. I need to make some changes as they gave me a small list of new improvements and also a few modifications. I'll make the changes, meaning I will remove the follow log or tail part because it can already be done with docker. I will keep the status part but improve it a little, and I’ll also work on exporting logs properly in TXT and JSON formats.
We wants to include these commands in inv dev.collect or inv demo.collect to start with, because the case i implemented with Docker Compose file is the least used part by clients. I’m trying to dive back into it as soon as possible

File: docker-compose.yml (hunk 1 above)
Comment (wartraxx51) [reply docker-compose.yml]: you are right

**Comment by wartraxx51** [comment]
added a debug variable to control the display of commands with print_cmd, as the previous code did not account for the boolean. By exporting the variable DEBUG=true, you can enable command output as well as visibility for hidden sections (the hide part)

**Review by BeArchiTek — APPROVED** [review]
LGTM

**Pull request merged**